cudapoa/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ cuda_add_library(cudapoa ${cga_library_type}
     src/batch.cpp
     )
 
-target_link_libraries(cudapoa cgautils cgalogging)
+target_link_libraries(cudapoa cgautils cgalogging cgaio)
```

Comment:
this was added in order to use fasta reader. But later the 6 fasta files were packed in a single text file, therefore `cgaio` is no longer needed and this change can be reverted.